rts/motoko-rts-tests/src/gc.rs

```diff
@@ -317,9 +325,13 @@ fn compute_reachable_objects(
 }
 
 fn check_continuation_table(mut offset: usize, continuation_table: &[ObjectIdx], heap: &[u8]) {
+    let table_addr = heap.as_ptr() as usize + offset;
     assert_eq!(read_word(heap, offset), TAG_ARRAY);
     offset += WORD_SIZE;
 
+    assert_eq!(read_word(heap, offset), make_pointer(table_addr as u32));
```

Comment:
Is this checking the forwarding pointer points directly at self? Why can't it have been forwarded properly at this point?

Reply:
Good point. This is just temporarily in this PR which does not yet use the incremental GC. It will be refined in the higher PR.